### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by mitas1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: go
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install libgl1-mesa-dev xorg-dev
```
Comment (mskrip) [inline .travis.yml]: Are the first two whitespaces necessary? This should be equivalent this in yml:

```yaml
before_install:
- sudo apt-get -qq update
- sudo apt-get install libgl1-mesa-dev xorg-dev
```

File: Makefile [hunk 2]
```diff
@@ -28,7 +28,7 @@ LDFLAGS=-ldflags '-X main.VERSION=${VERSION} -X main.COMMIT=${COMMIT} -X main.BR
 all: install goinstall
 
 $(BINDATA):
-	@go get -v github.com/jteeuwen/go-bindata
+	@go get -v github.com/jteeuwen/go-bindata/...
```
Comment (mskrip) [inline Makefile]: What do these ellipses do here? Wouldn't they install also all the dev dependencies of `go-bindata`? Or?

File: Makefile [hunk 3]
```diff
@@ -76,4 +76,7 @@ uninstall: clean
 run: goinstall
 	@${TARGET} &
 
+test:
+	@GOPATH=$(GOPATH) GOBIN=$(GOBIN) go test -v ./...
```
Comment (mskrip) [inline Makefile]: Do we still need this inline env `GOPATH` definition? Didn't we get rid of this?

File: Makefile (hunk 2 above)
Comment (mitas1) [reply Makefile]: See https://github.com/jteeuwen/go-bindata

File: .travis.yml (hunk 1 above)
Comment (mitas1) [reply .travis.yml]: Read something about yml format.

Comment (mskrip) [reply .travis.yml]: ok

File: Makefile (hunk 2 above)
Comment (mskrip) [reply Makefile]: nice

**Comment by mskrip** [comment]
Also adding caching would be useful

File: Makefile (hunk 3 above)
Comment (mitas1) [reply Makefile]: Nope. It is removed.

**Comment by mitas1** [comment]
I think I am done with this. Caching is not absolutely necessary here and we can add it later.

**Comment by mskrip** [comment]
LGTM

**Pull request merged**